src/FEValues/face_integrals.jl

```diff
@@ -1,11 +1,20 @@
 """
     face_to_element_transformation(point::Vec, ::Type{<:AbstractRefShape}, face::Int)
 
-Transform quadrature point from face's reference (N-1)D coordinates to ND coordinates on the
-cell's face.
+Transform quadrature point from face's reference coordinates to coordinates on the
+cell's face, increasing the number of dimensions by one.
 """
 face_to_element_transformation
 
+"""
+    element_to_face_transformation(point::AbstractVector, cell::AbstractCell{AbstractRefShape}, face::Int)
+
+Transform quadrature point from cell's coordinates to the face's reference coordinates, decreasing the number of dimensions by one.
+This is the inverse of `face_to_element_transformation`.
+This is the inverse of `face_to_element_transformation`.
+"""
+element_to_face_transformation
```

Comment:
They're tested in
https://github.com/Ferrite-FEM/Ferrite.jl/blob/ce1886d9e36058052f5f79cc7bbf3586eb21869d/test/test_interpolations.jl#L54-L70
I'm not sure if using spatial coordinates test will be helpful since IIUC it uses the cell corrdinates for both `CellValues` and `FaceValues`. Also, quadrature points already use these functions. Or you mean something else?

Reply:
That should be fine. :) I just missed this somehow